sdb/command.py

```diff
@@ -73,6 +74,50 @@ def help(cls, name: str):
             print("    {}".format(", ".join(cls.names)))
             print()
 
+        if cls.input_type is not None:
+            print("INPUT TYPE")
+            print(f"    This command primarily accepts "
+                  f"inputs of type {cls.input_type}.")
+            print()
+
+        if issubclass(cls, sdb.PrettyPrinter):
+            print("PRETTY PRINTER")
+            print(f"    This is a PrettyPrinter for {cls.input_type}.")
+            print(f"    If prints a human-readable decoding of the object.")
+            print(f"    For the raw object contents, pipe the "
+                  f"output to 'echo'.")
+            print()
+
+        if issubclass(cls, sdb.Walker):
+            print("PRETTY PRINTER")
+            print(f"    This is a Walker for {cls.input_type}.  "
+                  "See 'help walk'.")
```

Comment:
What information do we expect `help walk` to convey to the user for walkers? e.g. if I use `help avl`, and then `help walk`.. `help walk` doesn't really provide any more information to the user..

Do you think the lack of additional information is just because the docstring for `walk` is non-existent?

Reply:
I was thinking we should add the list of walkers, as well as a general description, to `help walk`.